### PR TITLE
Fix occlusion and overlap on welcome pages

### PR DESCRIPTION
**Pull request by EasyVector**
####  Description
Hello! I am the creator of issue #4212, and I have fixed those occlusions and overlaps on welcome pages. What is more, I also adjust the layouts of those other XML files with ConstraintLayout, just to keep consistency among those pages. I would genuinely appreciate it if you could kindly revise my code and leave me some advice. Thank you so much for your precious time!

#### Related issues
#4212 

#### Related PRs

There is no related PR.

#### Screenshots

Here are screenshots from PIxel 2 with API 29:

copy copy copy copy copy

#### Link to the automatically generated build APK

[app-obf-fdroid-debug.zip](https://github.com/openfoodfacts/openfoodfacts-androidapp/files/7100257/app-obf-fdroid-debug.zip)

### CODE REVIEW

File: app/src/main/res/layout/welcome_slide_1.xml [hunk 1]
```diff
@@ -8,36 +8,53 @@
...
         android:id="@+id/slide_2_image"
-        android:layout_width="@dimen/img_width_height"
-        android:layout_height="@dimen/img_width_height"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintWidth_percent="0.6"
+        app:layout_constraintDimensionRatio="2:1"
         android:layout_centerHorizontal="true"
-        app:srcCompat="@drawable/ic_nutriscore_c" />
+        android:contentDescription="TODO"
```
Comment (naivekook) [inline app/src/main/res/layout/welcome_slide_1.xml]: can we remove it and add suppress lint fields or replace it with `android:contentDescription="@null"` ?

File: app/src/main/res/layout/welcome_slide_1.xml [hunk 2]
```diff
@@ -8,36 +8,53 @@
...
-        android:layout_below="@id/slide_2_image"
-        android:layout_centerHorizontal="true"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:gravity="center_horizontal"
         android:padding="20dp"
         android:text="@string/title_welcome_nutriscore"
         android:textColor="@android:color/white"
-        android:textSize="20sp"
+        android:textSize="20dp"
```
Comment (naivekook) [inline app/src/main/res/layout/welcome_slide_1.xml]: move this value to dimens please

**Comment by naivekook** [comment]
@yuhuitech Hi! Thanks for your contribution! 
Can you also tweak the page indicators to fit the container?

**Comment by EasyVector** [comment]
OK, I will be on it. :)

**Comment by EasyVector** [comment]
Hello! I think I have fixed the blocked page indicators. Here is the screenshot after my fix:

copy

**Review by naivekook — APPROVED** [review]
lgtm

**Comment by teolemon** [comment]
Thank you @yuhuitech, merging

**Comment by VaiTon** [comment]
Thank you @yuhuitech !

**Pull request merged**